public/scripts/socket-github.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Client-side Event Handling for deploy/github
+ */
+$(function () {
+  var socket = io();
+  socket.emit('ghpages-deploy', {
+    'email': "#{email}",
```

Comment:
I think this won't render. we have to think something else about this. right now we going in monolithic fashion. If we separate front-end and backend. we can achieve maintainable code. since we shifting the yaydoc to a proper CI platform. I think we have to move from monolithic.